{"uuid": "08a5633f-ee75-451c-9b61-96e5de790136", "children": ["a6b5fa5c-5cbd-4dd1-9a11-876a015c0689"], "befores": [{"name": "_xunit_setup_class_fixture_Test_Commerce", "status": "passed", "start": 1673596897632, "stop": 1673596903347}], "afters": [{"name": "_xunit_setup_class_fixture_Test_Commerce::0", "status": "broken", "statusDetails": {"message": "AttributeError: type object 'Test_Commerce' has no attribute 'self'\n", "trace": "  File \"/Users/a00/Library/Python/3.8/lib/python/site-packages/allure_commons/_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"/Users/a00/Library/Python/3.8/lib/python/site-packages/_pytest/fixtures.py\", line 916, in _teardown_yield_fixture\n    next(it)\n  File \"/Users/a00/Library/Python/3.8/lib/python/site-packages/_pytest/python.py\", line 868, in xunit_setup_class_fixture\n    _call_with_optional_argument(func, self.obj)\n  File \"/Users/a00/Library/Python/3.8/lib/python/site-packages/_pytest/python.py\", line 786, in _call_with_optional_argument\n    func(arg)\n  File \"/Users/a00/PycharmProjects/pythonProject3/ddt/Commerce_test.py\", line 14, in teardown_class\n    self.self.quit()\n"}, "start": 1673596903368, "stop": 1673596907399}], "start": 1673596897632, "stop": 1673596907399}